src/extensions/surveys.tsx

```diff
@@ -219,3 +118,752 @@ export function generateSurveys(posthog: PostHog) {
         callSurveys(posthog, false)
     }, 3000)
 }
+
+const defaultBackgroundColor = '#eeeded'
+
+export function QuestionHeader({
+    question,
+    description,
+    backgroundColor,
+}: {
+    question: string
+    description?: string | null
+    backgroundColor?: string
+}) {
+    return (
+        <div style={{ backgroundColor: backgroundColor || defaultBackgroundColor }}>
+            <div className="survey-question">{question}</div>
```

Comment:
I understand this is out of the scope of this PR but for long term code quality it would be nice to have a solid base of BEM style classes rather than random simple names.

Not expecting this to happen now but follow up ideas...